src/CommunityToolkit.Maui.Core/Views/DrawingView/DrawingLine.shared.cs

```diff
@@ -0,0 +1,67 @@
+﻿using System.Collections.ObjectModel;
+
+namespace CommunityToolkit.Maui.Core.Views;
+
+/// <summary>
+/// The Line object is used to describe the lines that are drawn on a <see cref="IDrawingView"/>.
+/// </summary>
+public class DrawingLine
```

Comment:
what's the difference between this and `MauiDrawingLine`?

Reply:
DrawingLine is a Virtual Control (uses Microsoft.Maui.Graphics.Color etc),

MauiDrawingLine is a Platform Control (uses Android.Graphics.Color etc)